lib/puppet/configurer.rb

```diff
@@ -296,50 +296,11 @@ def run_internal(options)
       configured_environment = Puppet[:environment] if Puppet.settings.set_by_config?(:environment)
 
       # We only need to find out the environment to run in if we don't already have a catalog
-      unless (cached_catalog || options[:catalog] || Puppet[:strict_environment_mode])
-        begin
-          node = nil
-          node_retr_time = thinmark do
-            node = Puppet::Node.indirection.find(Puppet[:node_name_value],
-              :environment => Puppet::Node::Environment.remote(@environment),
-              :configured_environment => configured_environment,
-              :ignore_cache => true,
-              :transaction_uuid => @transaction_uuid,
-              :fail_on_404 => true)
-          end
-          options[:report].add_times(:node_retrieval, node_retr_time)
```

Comment:
This report metric will go away, should we keep adding it for backwards compatibility or work with the PE console team to remove it altogether?

Reply:
I'm pretty sure they just display whatever metrics are present. But to minimize breakage, I'd be fine setting the metric to 0 always.

Reply:
I'll test this, would prefer not adding it if possible.

Reply:
This is without us sending the metric, I think we're good:
![image](https://user-images.githubusercontent.com/25789827/123791345-ff778500-d8e7-11eb-87bc-7d7bac9bbd0d.png)

At some point we should probably remove it from the console as well.